electron/secureStorage/service.js

```diff
@@ -0,0 +1,57 @@
+import { ipcMain } from 'electron'
+import config from 'config'
+import { mainLog } from '@zap/utils/log'
+import createStorage from '@zap/utils/secureStorage'
+
+/**
+ * createStorageService - Create secure storage service.
+ *
+ * @param  {object} mainWindow Browser window
+ */
+export default function createStorageService(mainWindow) {
+  const storage = createStorage(config.secureStorage.namespace)
+  // helper func to send messages to the renderer process
+  const send = (msg, params) => mainWindow.webContents.send(msg, params)
+  const key = 'app-password'
```

Comment:
Rather than having a hardcoded key here, what if we extracted this so that this whole thing was more reusable:

So rather than `getPassword` you'd have `get(key)`. Rather than `deletePassword` you would have `delete(key)` etc.

We still don't know how we will store additional sensitive data (new keys under the store, in IDB encrypted with the password etc), but it would probably be good to make this method more generic anyway so that we have an easy way to manipulate multiple things in the key store.

We can defer this to a follow up refactor PR if you prefer though.